influxql/parser.go

```diff
@@ -864,14 +864,32 @@ func (p *Parser) parseShowFieldKeysStatement() (*ShowFieldKeysStatement, error)
 // This function assumes the "DROP SERIES" tokens have already been consumed.
 func (p *Parser) parseDropSeriesStatement() (*DropSeriesStatement, error) {
 	stmt := &DropSeriesStatement{}
+	var err error
 
-	// Read the name of the series to drop.
-	lit, err := p.parseIdent()
-	if err != nil {
+	tok, _, _ := p.scanIgnoreWhitespace()
+	if tok == FROM {
```

Comment:
nit:  you can save a line of code...
`if tok, _, _ := p.scanIgnoreWhitespace(); tok == FROM {`